src/toil_lib/tools/variant_annotation.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+from toil_lib.programs import docker_call
+
+
+def gatk_genotype_gvcfs(job, gvcf_ids, config, emit_threshold=10.0, call_threshold=30.0):
+    """
+    Runs GenotypeGVCFs on one or more gVCFs generated by HaplotypeCaller.
+
+    :param JobFunctionWrappingJob job: Toil Job instance
+    :param dict gvcf_ids: Dictionary of GVCF FileStoreIDs
+    :param Namespace config: Configuration options and shared FileStoreIDs
+    :param float emit_threshold: Minimum phred-scale confidence threshold for
+                                 a variant to be emitted. Default: 10.0
+    :param float call_threshold: Minimum phred-scale confidence threshold for
+                                 a variant to be called. Default: 30.0
+    :return: VCF FileStoreID
+    :rtype: str
+    """
+    job.fileStore.logToMaster('Running GATK GenotypeGVCFs')
+
+    inputs = {'genome.fa': config.genome_fasta,
+              'genome.fa.fai': config.genome_fai,
+              'genome.dict': config.genome_dict}
+    inputs.update(gvcf_ids)
```

Comment:
Why not just have all of the IDs be in the config since you're going to require one anyway? What's the reason for having a separate dict just for `gvcf_ids`?